internal/fourslash/tests/basicInterfaceMembers_test.go

```diff
@@ -0,0 +1,54 @@
+package fourslash_test
+
+import (
+	"testing"
+
+	"github.com/microsoft/typescript-go/internal/fourslash"
+	"github.com/microsoft/typescript-go/internal/ls"
+	"github.com/microsoft/typescript-go/internal/lsp/lsproto"
+	"github.com/microsoft/typescript-go/internal/testutil"
+)
+
+func TestBasicInterfaceMembers(t *testing.T) {
+	t.Parallel()
+	defer testutil.RecoverAndFail(t, "Panic on fourslash test")
+	const content = `export {};
+interface Point {
+	x: number;
+	y: number;
+}
+declare const p: Point;
+p./*a*/`
+	f, done := fourslash.NewFourslash(t, nil /*capabilities*/, content)
+	defer done()
```

Comment:
Rather than returning a `done` func to defer, you can just use `t.Cleanup()` from inside `NewFourslash`, which will tell the testing package to run it after the test completes.